src/test/fuzz/fuzz.h

```diff
@@ -5,10 +5,39 @@
 #ifndef BITCOIN_TEST_FUZZ_FUZZ_H
 #define BITCOIN_TEST_FUZZ_FUZZ_H
 
-#include <stdint.h>
+#include <attributes.h>
+#include <optional.h>
+#include <serialize.h>
```

Comment:
This header takes a long time to parse, but not every fuzz target is using it. It might reduce compile time to put the fuzzedDataProvider helpers in a new module (header) and keep this header the raw and minimal "main fuzz header"

Reply:
Good point. Now fixed.